ergotree-ir/Cargo.toml

```diff
@@ -30,6 +30,9 @@ base16 = "0.2.1"
 num-bigint = "0.4.0"
```

Comment:
I wonder if we still need an explicit dep on `num-bigint` since `num256` pulls it already.

Reply:
I tried removing it and got 2 errors: in `bigint256.rs` and in `dlog_group.rs` (which needs `num_bigint::Sign`).